includes/admin/upgrades/class-give-updates.php

```diff
@@ -294,7 +294,7 @@ public function __redirect_admin() {
 		if (
 			! wp_doing_ajax() &&
 			current_user_can( 'manage_give_settings' ) &&
-			Give_Cache_Setting::get_option( 'give_show_db_upgrade_complete_notice' ) &&
+			get_option( 'give_show_db_upgrade_complete_notice' ) &&
```

Comment:
This is ain fix  in this pr and remaining is code formatting which occurs because of PHPCS